### DISCUSSION TITLE
feat: FolderTree supports different icons according to file extensions.

### DISCUSSION THREAD

**Discussion by joel-jo-querypie** [DTStack/molecule#677] [Ideas]
### Description

![Screen Shot 2022-01-22 at 9 50 47 PM](https://user-images.githubusercontent.com/44131043/150639219-a75a7180-2653-4e08-a223-a442321c579e.jpg)

They have different file extensions. But everyone has the same icon. It is not easy to distinguish files. Showing different icons according to file extensions can make it easier to distinguish files.

### Solution
Identify the file extension from the title. And render the icon with this information.

**Comment by joel-jo-querypie** [comment]
is it okay to send PR using a branch with the name `feat/FolderTreeIcon`  ??

**Comment by wewoor** [comment]
@guymoon Hi, guymoon, we are welcome every commit from the community! The` feat/FolderTreeIcon` branch name is ok~

You can PR your updates here, and then we can discuss it~
